src/tokenizer/index.js

```diff
@@ -663,9 +664,17 @@ export default class Tokenizer extends LocationParser {
       isFloat = true;
     }
 
+    if (this.hasPlugin("bigint")) {
+      if (next === 110) { // 'n'
+        if (isFloat) this.raise(start, "Invalid BigIntLiteral");
+        ++this.state.pos;
+        isBigInt = true;
+      }
+    }
+
     if (isIdentifierStart(this.fullCharCodeAtPos())) this.raise(this.state.pos, "Identifier directly after number");
 
-    const str = this.input.slice(start, this.state.pos).replace(/_/g, "");
+    const str = this.input.slice(start, this.state.pos).replace(/[_n]/g, "");
```

Comment:
Is it worth putting a comment here to explain we are removing the _ for numeric separators and n for BigInt?